config-connector/solutions/apps/helm/wordpress/README.md

```diff
@@ -0,0 +1,55 @@
+# Wordpress
+
+==================================================
+
+## NAME
+
+  wordpress
+
+## SYNOPSIS
+
+The WordPress application demonstrates how you can configure a WordPress site powered by GCP MySQL database and using Workload Identity for authentication.
+
+## CONSUMPTION
+
+  ```bash
+   helm install ./charts/wordpress-gcp/ --set google.projectId=[PROJECT_ID]
+  ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector and [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster).
+1. [Helm](https://helm.sh/docs/using_helm/)
```

Comment:
We might also want to mention what APIs customers need to enable before they install the charts. For example, I failed to install the charts for the first time because I didn't have Cloud SQL Admin API enabled.

Reply:
Thanks! will fix with the next change.

Reply:
Added serviceusage resource